src/app/map/Tile/Item/content.tsx

```diff
@@ -182,7 +186,7 @@ const DescriptionSection = ({ description, scale, tileId }: { description: strin
 
 };
 
-const UrlSection = ({ url, scale }: { url: string; scale: TileScale }) => {
+const UrlSection = ({ url, scale, textClasses }: { url: string; scale: TileScale; textClasses: string }) => {
```

Comment:
I think this is dead code that you can delete. Instead when there is a url, make the title a link to this url